src/main/java/client/PetClient.java

```diff
@@ -2,69 +2,24 @@
 
 import enums.EStatus;
 import dto.requests.pet.Pet;
-import dto.requests.pet.ResponseInfo;
+import dto.requests.ResponseInfo;
 import io.qameta.allure.Step;
 import io.restassured.RestAssured;
-import io.restassured.builder.RequestSpecBuilder;
-import io.restassured.builder.ResponseSpecBuilder;
-import io.restassured.http.ContentType;
 import io.restassured.response.Response;
 import io.restassured.specification.RequestSpecification;
-import io.restassured.specification.ResponseSpecification;
 import org.apache.http.HttpStatus;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 import java.io.File;
-import java.io.FileInputStream;
 import java.io.IOException;
 import java.util.*;
 
-public class PetClient {
-    final private static String BASE_URI = "https://petstore.swagger.io";
+public class PetClient extends Client {
     final private static String PET_ENDPOINT = "/v2/pet/";
     final private static String PET_FIND_BY_STATUS_ENDPOINT = PET_ENDPOINT + "findByStatus?status=";
-    final private static String PET_IMAGE_UPLOAD_ENDPOINT = PET_ENDPOINT + "%s/uploadImage";    // setting here pet ID
+    final private static String PET_IMAGE_UPLOAD_ENDPOINT = PET_ENDPOINT + "%s/uploadImage";    // setting here petStoreTests.pet ID
 
-    public static final Logger LOGGER = LoggerFactory.getLogger(PetClient.class);
 
-    @Step("Building request specification")
-    public static RequestSpecification buildReq() throws IOException {
-        LOGGER.debug("building request specification ");
-
-        return new RequestSpecBuilder()
-            .setBaseUri(BASE_URI)
-            .setContentType(ContentType.JSON)
-            .build();
-    }
-
-    @Step("Building request specification")
-    public static RequestSpecification buildUncheckedReq() throws IOException {
-        LOGGER.debug("building request specification ");
-
-        return new RequestSpecBuilder()
-            .setBaseUri(BASE_URI)
-            .build();
-    }
-
-    @Step("Building response specification")
-    public static ResponseSpecification buildRes() {
-        LOGGER.debug("building response specification ");
-
-        return new ResponseSpecBuilder()
-            .expectContentType(ContentType.JSON)
-            .build();
-    }
-
-    @Step("Building response specification without any response checking")
-    public static ResponseSpecification buildUncheckedRes() {
-        LOGGER.debug("building response specification ");
-
-        return new ResponseSpecBuilder()
-            .build();
-    }
-
-    @Step("Adding pet to the store")
+    @Step("Adding petStoreTests.pet to the petStoreTests.store")
```

Comment:
What is the significance of `petStoreTests` ?

Reply:
oh it was done by refactor, I will do fix this